src/cache/cache_las.c

```diff
@@ -18,6 +18,28 @@
 	(WT_SESSION_IGNORE_CACHE_SIZE | WT_SESSION_READ_WONT_NEED |	\
 	WT_SESSION_NO_RECONCILE)
 
+/*
+ * __las_set_read_uncommitted --
+ *	Switch to read-uncommitted.
+ */
```

Comment:
I'd prefer these be balanced (in naming) and both general even if we only have one usage at the moment. How about `las_set_isolation(session, iso, *saved)` and then `las_restore_isolation(session, saved)`?

Reply:
Done in ea154c9.